docker/py3-native/Dockerfile

```diff
@@ -0,0 +1,103 @@
+FROM devdemisto/py3-tools-ubi:1.0.0.37715
+
+COPY requirements.txt .
+COPY chromium/download_chromedriver.sh .
+COPY repos /repos
+
+# handles chromium + tesseract
```

Comment:
Are all these comments needed?

Reply:
no they can be removed before merge, these are just for making things easier for me when to follow up.